packages/shared/src/components/modals/common/ModalClose.tsx

```diff
@@ -13,7 +13,8 @@ function ModalCloseComponent(
       {...props}
       onClick={onClick}
       ref={ref}
-      className={classNames('right-4 z-1', className)}
+      size="medium"
```

Comment:
I think we can just use the `buttonSize` instead of introducing a new one.

Reply:
Yeap, sharp eyes 👀